controllers/tutorialsControllers.js

```diff
@@ -21,6 +22,7 @@ const createTutorial = async (req, res) => {
     published: req.body.published,
   });
   try {
+    await schema.validateAsync(req.body);
```

Comment:
Did you test by passing invalid title or any other fields? What's the response you're getting?